azure-quantum/tests/unit/aio/test_aio_job.py

```diff
@@ -29,7 +29,7 @@
     functools.partial(microsoft.SimulatedAnnealing, beta_start=0),
     functools.partial(microsoft.ParallelTempering, sweeps=100),
     functools.partial(microsoft.Tabu, sweeps=100),
-    functools.partial(microsoft.QuantumMonteCarlo, trotter_number=1),
+    functools.partial(microsoft.QuantumMonteCarlo, trotter_number=4),
```

Comment:
Do you know if we have to update the test recordings as well (cc: @ricardo-espinoza)? I think you can regenerate if you delete the recordings folder and then re-run tests, but I'm not super familiar with this part of code-base.

Reply:
There 6 other methods which had similar changes like this one (Switch from QIO to QIOTE, like test_job_submit_microsoft_parallel_tempering). If changes to recording would be needed, they would fail.

Reply:
@xinyi-joffre we had a similar question for another PR too, I believe it is fine for this one - but I will start a thread to discuss